packages/core/useSpeechRecognition/index.ts

```diff
@@ -120,10 +120,17 @@ export function useSpeechRecognition(options: UseSpeechRecognitionOptions = {})
       if (newValue === oldValue)
         return
 
-      if (newValue)
-        recognition!.start()
-      else
-        recognition!.stop()
+      try {
+        if (newValue) {
+          recognition!.start()
+        }
+        else {
+          recognition!.stop()
+        }
+      }
+      catch (err: unknown) {
+        error.value = err as unknown as SpeechRecognitionErrorEvent
```

Comment:
is this error always `SpeechRecognitionErrorEvent`?

Reply:
no, in this situation it will be `DOMException`, I set it to `Error` so that it could include other errors.